tests/test_api_hosts_update.py

```diff
@@ -271,6 +271,25 @@ def test_patch_produces_update_event_no_insights_id(
     )
 
 
+def test_patch_by_namespace_produces_update_event(
+    event_producer_mock, event_datetime_mock, db_create_host, db_get_host, api_patch
+):
+    host = db_host()
+    created_host = db_create_host(host=host, extra_data={"facts": DB_FACTS})
```

Comment:
This is an optional change, but since `host` is only used once, I personally wouldn't create a variable for it. Up to you, though! If you think it makes it less readable, I'm fine with leaving it the way it is.
```suggestion
    created_host = db_create_host(host=db_host(), extra_data={"facts": DB_FACTS})
```